test/buildd/test_dist.py

```diff
@@ -73,40 +60,47 @@ def create_package_leftovers(
     (test_project_dir_path / f'{project_name}.egg-info' / 'leftover').touch()
 
 
-def create_dags_leftovers(test_project_dir_path: Path = TEST_PROJECT_PATH):
+def create_dags_leftovers(test_project_dir_path=None):
+    test_project_dir_path = test_project_dir_path or TEST_PROJECT_PATH
     mkdir(test_project_dir_path / '.dags')
     (test_project_dir_path / '.dags' / 'leftover').touch()
 
 
-def dags_leftovers_exist(test_project_dir_path: Path = TEST_PROJECT_PATH):
+def dags_leftovers_exist(test_project_dir_path=None):
```

Comment:
Why?

Reply:
Now 'TEST_PROJECT_PATH' is different for each test (tmp directory) and redefined in `TestCase.setUp`